artview/core/common.py

```diff
@@ -160,9 +164,12 @@ def display_data(self, data):
             the table """
 
         if len(data)==0:
-            data = ["No data for selected ROI."]
+            data = [["No data for selected ROI."]]
+            nrows = 0
+            ncols = 0
+        else:
+            nrows, ncols = data.shape[0], data.shape[1]
```

Comment:
Does this occur if you select outside of plot region?
If you select inside plot region, there is still data, correct?